src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTokenExtensions.vb

```diff
@@ -57,6 +57,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsGlobalStatementContext(token As SyntaxToken, position As Integer) As Boolean
```

Comment:
I don't understand what it means for a token to be a "global statement context".  Is the token *in* the context?